README.md

```diff
@@ -1,28 +1,26 @@
-# OER-StöberSpecs
+# StöberSpecs – Werkzeuge und Verfahren zur Standardisierung von OER-Metadaten
 
-## Werkzeuge und Verfahren zur Standardisierung von OER-Metadaten
+## Die DINI AG KIM als Plattform zur Spezifikation von Metadatenstandards im Bildungsbereich
 
-Ausgestattet mit Fördermitteln von Bund und Ländern beschäftigen sich immer mehr Akteure und Projekte mit dem Aufbau von Infrastrukturen zur Erstellung, Publikation und zum Teilen von [Open Educational Resources (OER)](https://de.wikipedia.org/wiki/Open_Educational_Resources). Innerhalb einer solchen Infrastruktur spielen Metadaten – strukturierte Beschreibungen, die Informationen über bestimmte Merkmale der beschriebenen Dinge erfassen – eine zentrale Rolle.
+Ausgestattet mit Fördermitteln von Bund und Ländern beschäftigen sich immer mehr Akteure und Projekte im Schul- und Hochschulbereich mit dem Aufbau von Infrastrukturen zur Erstellung, Publikation und zum Teilen von [Open Educational Resources (OER)](https://de.wikipedia.org/wiki/Open_Educational_Resources). Innerhalb einer solchen Infrastruktur spielen Metadaten – strukturierte Beschreibungen, die Informationen über bestimmte Merkmale der beschriebenen Dinge erfassen – eine zentrale Rolle.
```

Comment:
In der Überschrift wurde "OER-Metadaten" auf "Metadatenstandards im Bildungsbereich" generalisiert. Soll diese Generalisierung hier fortgesetzt werden? I.S.v. "[...] Publikationen und zum Teilen von Bildungsmaterialien, insbesondere Open Educational Resources..."